apply/helpers.py

```diff
@@ -68,6 +72,33 @@ def extract_subset_of_data_from_application(application_data: dict, data_subset_
     return application_data[data_subset_name]
 
 
+def get_change_requests(application_id: str) -> dict[str, list]:
+    assessment_flags = (
+        db.session.query(AssessmentFlag)
+        .join(FlagUpdate)
+        .filter(
+            and_(
+                AssessmentFlag.application_id == application_id,
+                FlagUpdate.status == FlagStatus.RAISED,
+            )
+        )
+        .all()
+    )
+
+    if not assessment_flags:
+        return {}
+
+    assessor_change_requests: dict[str, list] = {}
+    for change_request in assessment_flags:
+        for field_id in change_request.field_ids:
+            if field_id not in assessor_change_requests:
+                assessor_change_requests[field_id] = []
+
+            assessor_change_requests[field_id].extend([update.justification for update in change_request.updates])
+
+    return assessor_change_requests
+
+
 def format_rehydrate_payload(
```

Comment:
This method currently receives a bunch of stuff which has been fetched from backends alreday and describes itself as formatting it in a way that can be consumed by the form runner. 

Extending it to also fetch data will work fine but might make understanding/ testing it a bit confusing if you're getting some of the data outside, so of the data in here etc.

I don't think this is a blocker as I appreciate the route handlers that call this method are hundreds of lines long already, just conscious of spreading future us knowing where to look and what patterns to follow

Reply:
Passing change_requests as a parameter to the function. Good call.